libs/agno/agno/tools/openai.py

```diff
@@ -9,14 +14,39 @@
 
 client = OpenAIClient()
 
+# Define Literal types for allowed OpenAI parameter values
+OpenAIImageSize = Literal["256x256", "512x512", "1024x1024", "1792x1024", "1024x1792"]
+OpenAIVoice = Literal["alloy", "echo", "fable", "onyx", "nova", "shimmer"]
+OpenAITTSModel = Literal["tts-1", "tts-1-hd"]
+OpenAITTSFormat = Literal["mp3", "opus", "aac", "flac", "wav", "pcm"]  # Aligned with client expectation
+
 
 class OpenAITools(Toolkit):
     """Tools for interacting with OpenAIChat API"""
 
-    def __init__(self, **kwargs):
+    def __init__(
+        self,
+        enable_transcription: bool = True,
+        enable_image_generation: bool = True,
+        enable_speech_generation: bool = True,
+        default_tts_voice: OpenAIVoice = "alloy",
+        default_tts_model: OpenAITTSModel = "tts-1",
+        default_tts_format: OpenAITTSFormat = "mp3",
```

Comment:
```suggestion
        text_to_speech_voice: OpenAIVoice = "alloy",
        text_to_speech_model: OpenAITTSModel = "tts-1",
        text_to_speech_format: OpenAITTSFormat = "mp3",
```

These params are set by humans, they should be easy to understand and well documented (add docs please)